src/frontend/js/components/Modal/index.tsx

```diff
@@ -1,13 +1,31 @@
-import { PropsWithChildren, useMemo } from 'react';
+import { ReactNode, useMemo } from 'react';
 import ReactModal from 'react-modal';
+import { defineMessages, FormattedMessage, useIntl } from 'react-intl';
+
+const messages = defineMessages({
+  closeDialog: {
+    defaultMessage: 'Close modal',
```

Comment:
For non technical user, I believe dialog is more appropriate than modal (or we should use `dialog modal` 😅)
```suggestion
    defaultMessage: 'Close dialog',
```